### PR TITLE
Update Doozy with base layer loading screen

### PR DESCRIPTION
**Pull request by m4rtint**
Assets/Script/Managers/AudioManager.cs
Assets/Script/Managers/PlayerManager.cs

Only files I've changed for code.

### CODE REVIEW

File: Assets/Script/Managers/PlayerManager.cs [hunk 1]
```diff
@@ -143,7 +143,7 @@ private void InitializedCarSpawnPoints()
 
     private void CreatePlayer(PlayerInformation playerInfo, Player playerNumber)
     {
-        PlayerController playerController = Instantiate(m_playerController_Template.gameObject).GetComponent<PlayerController>();
+        PlayerController playerController = Instantiate(m_playerController_Template.gameObject, transform).GetComponent<PlayerController>();
```
Comment (Gursaanj) [inline Assets/Script/Managers/PlayerManager.cs]: This makes the playerController GameObject a child of the PlayManager right? Any reason for doing this. or is it just for neatness sake?

Comment (m4rtint) [reply Assets/Script/Managers/PlayerManager.cs]: Yes. 
So the flow currently is
MainMenu Loaded + Active -> Generic UI + Map Loaded/Inactive

Generic UI + Map Loaded/Active
Players are created in the Main Menu Scene (it's still the first scene)

Unload Main Menu -> Players get unloaded

File: Assets/Doozy/Engine/Scripts/Coroutiner.cs [hunk 2]
```diff
@@ -45,6 +45,14 @@ public static Coroutiner Instance
 
         #region Unity Methods
 
+        #if UNITY_2019_3_OR_NEWER
+        [RuntimeInitializeOnLoadMethod(RuntimeInitializeLoadType.SubsystemRegistration)]
+        private static void RunOnStart()
```
Comment (wbeltran777) [inline Assets/Doozy/Engine/Scripts/Coroutiner.cs]: What's the purpose of having this function in three different classes? If it's private static does it mean that they're all modifying the same ApplicationIsQuitting bool?

Comment (m4rtint) [reply Assets/Doozy/Engine/Scripts/Coroutiner.cs]: I did not write this.

Comment (wbeltran777) [reply Assets/Doozy/Engine/Scripts/Coroutiner.cs]: Alrighty! Then everything looks great hahahah

**Pull request merged**